.github/workflows/ci.yaml

```diff
@@ -80,9 +80,71 @@ jobs:
         name: coverage
         path: "${{ env.COVERAGE_FILE }}"
 
+  pathogen-ci:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    env:
+      repodata_use_zst: true
+    strategy:
+      max-parallel: 4
+      matrix:
+        include:
+          - { pathogen: avian-flu, build-args: auspice/flu_avian_h5n1_ha.json }
+          - { pathogen: ebola }
+          - { pathogen: lassa }
+          - { pathogen: monkeypox }
+          - { pathogen: mumps }
+          - {
+              pathogen: ncov,
+              build-args: all_regions -j 2 --profile nextstrain_profiles/nextstrain-ci,
+            }
+          - {
+              pathogen: seasonal-flu,
+              build-args: --configfile profiles/ci/builds.yaml -p,
+            }
+          - { pathogen: zika }
```

Comment:
As noted in https://github.com/nextstrain/docker-base/commit/12000a20d9df02b59cce4aa7f466833fb54df362, I think it's unfortunate that we need to include `build-args` here and am a bit disappointed to see it continue proliferating.

Compounding that, I think it's unfortunate that this doesn't use our existing pathogen-repo-ci workflow (or some alternative equivalent). I understand it can't be used as-is and would need modifications and that's a barrier, but I think it'd be important to pursue modifications sooner than later rather than leaving Augur with this bespoke version that's certain to drift out of date. It doesn't seem like you have plans to circle back to it, though?